bridge/tracker/rootchain.go

```diff
@@ -0,0 +1,231 @@
+package tracker
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/0xPolygon/polygon-edge/blockchain/storage"
+	"github.com/0xPolygon/polygon-edge/blockchain/storage/leveldb"
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/hashicorp/go-hclog"
+	"github.com/umbracle/go-web3"
+	"github.com/umbracle/go-web3/abi"
+	client "github.com/umbracle/go-web3/jsonrpc"
+	"math/big"
+	"os"
+	"path/filepath"
+)
+
+const (
+	//	required block depth for fetching events on the rootchain
+	DefaultBlockConfirmations = 6
+)
+
+//	contractABI is used to create query filter
+//	that matches events defined in a smart contract.
+type contractABI struct {
+	//	address of smart contract
+	address web3.Address
+
+	// signatures of events defined in smart contract
+	events []*abi.Event
+}
```

Comment:
What do you think about renaming this to `eventABI`, since you're storing event signatures here, and the address of the actual smart contract that emits them?
Having it be `contactractABI` is misleading, as you don't keep any ABI data concerning the contract apart from the events.

Reply:
[Renamed](https://github.com/0xPolygon/polygon-edge/pull/386/commits/a2f94e582faa1d97c74faa41bd203adef489fc54)